src/components/Cloud/Environment-Data.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DashboardBlock from '../Dashboard-Block';
+
+/**
+ * Data about the current environment's data.
+ *
+ * @param {Object} gitData        Data about the current Git state in this environment.
+ * @param {Object} environmentData Data about the HM Cloud environment in this environment.
+ */
+const EnvironmentData = ( { gitData, environmentData } ) => <DashboardBlock title="Application Data" id="environment-data">
+	<div className="environment-data">
+		<div className="environment-data-block">
+			<h3 className="environment-data-block__title">Application Version</h3>
+			<dl>
+				<dt>Git Branch:</dt>
+				<dd>{ gitData.branch }</dd>
+				<dt>Commit:</dt>
+				<dd>{ gitData.commit && `${ gitData.commit.description } (${ gitData.commit.rev.substring( 0, 7 ) })` }</dd>
+			</dl>
+		</div>
+		<div className="environment-data-block">
+			<h3 className="environment-data-block__title">HM Cloud Version</h3>
```

Comment:
Really just a question of how we name this stuff here. Maybe `HM Cloud Software`